pages/agent/v3/hooks.md.erb

```diff
@@ -68,7 +68,7 @@ The following is a complete list of available hooks, and the order in which they
     <tr>
       <td style="white-space: nowrap"><code>command</code></td>
       <td style="white-space: nowrap">Plugin, Repository, Agent</td>
-      <td>Overrides the default command running behavior. <em>Note:</em> If multiple command hooks are found, only the first will be run.</td></tr>
+      <td>Overrides the default command running behavior. If multiple command hooks are found, only the first will be run. Each pipeline must have a command hook available in at least one of the locations (plugin, repository, or agent)</td></tr>
```

Comment:
Clarifying this is :+1: :+1: 

> Each pipeline must have a command hook available in at least one of the locations (plugin, repository, or agent

But there is something niggling at me with the sentence. How about

```suggestion
      <td>Overrides the default command running behavior. If multiple command hooks are found, only the first will be run. A pipeline must have a command hook defined defined in the plugin, the repository or the agent.</td></tr>
```

Although, you can just define the command in the pipeline as well, 

```suggestion
      <td>Overrides the default command running behavior. If multiple command hooks are found, only the first will be run. A pipeline must have a command hook defined in one or more of the pipeline, the plugin hook, the repository hook, or the agent hook.</td></tr>
```

Reply:
As I understand it, in the pipeline, you could say which command to run, but it would look for the command in either a plugin, agent, or the repo? You don't actually write commands in the pipeline yaml? And there's no pipeline directory setup with command files that I just haven't come across?

Reply:
I'm also really not sure "defined" is the right word? That suggests you set it in a config/describe it somewhere? But you literally just stick a command script in the hooks directory?